docs/user-guide/walkthrough/README.md

```diff
@@ -134,7 +134,7 @@ For this example, we'll be creating a Redis pod, with a named volume and volume
       emptyDir: {}
   ```
 
-1. Define a volume mount within a container definition:
+2. Define a volume mount within a container definition:
```

Comment:
It doesn't correctl recognize that this is the same list in the generated HTML; http://kubernetes.io/v1.0/docs/user-guide/walkthrough/README.html#volumes